src/app.js

```diff
@@ -1,21 +1,41 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+
+  const pathname = normalizedUrl.pathname;
+  const parts = pathname.split('/').filter(part => part !== '');
+
+  const query = {};
+
+  if (normalizedUrl.searchParams) {
+    for (const [key, value] of normalizedUrl.searchParams.entries()) {
```

Comment:
U use construction `normalizedUrl.searchParams` twice - here and in if. Make sense create variable for that expression.